docs/source/models/supported_models.rst

```diff
@@ -131,6 +131,10 @@ Alongside each architecture, we include some popular models that use it.
     - Orion
     - :code:`OrionStarAI/Orion-14B-Base`, :code:`OrionStarAI/Orion-14B-Chat`, etc.
     -
+  * - :code: `PaliGemmaForConditionalGeneration`
```

Comment:
```suggestion
  * - :code:`PaliGemmaForConditionalGeneration`
```

Reply:
Will update this PR with the VLM doc PR when it's merged